modules/ROOT/pages/box/box-connector-config-topics.adoc

```diff
@@ -4,42 +4,42 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../../assets/images/
 
-== Configure the Box Connector Global Element
+To use the Box connector in a Mule application, configure a global Box configuration element that can be used by as many Box connectors as needed for your application.
 
-To use the Box connector in a Mule application, configure a global Box configuration element that can be used by as many Box connectors as
-needed for your application.
+== Configure the Box Connector Global Element
 
 . Click the **Global Elements** tab at the base of the canvas.
-. In the Global Configuration Elements screen, click **Create**.
-. In the Choose Global Type wizard, expand Connector Configuration and select **Mule-box-connector Config** and click **Ok**.
-The configuration screen for `OAUTH 2.0` with Authorization Code, grant type displays. This is the only configuration that Box connector supports.
+. In the *Global Configuration Elements* screen, click *Create*.
+. In the *Choose Global Type* wizard, expand Connector Configuration and select *Mule-box-connector Config* and click *OK*.
+. In the configuration screen for `OAUTH 2.0` with Authorization Code, grant type displays. This is the only configuration that Box connector supports.
```

Comment:
This doesn't seem like a step. It seems like a step result.  I don't see the user taking any action here.  ?